src/printer.js

```diff
@@ -524,9 +524,7 @@ function printExpression(path, options, print) {
             path.map(valuePath => {
               const node = valuePath.getValue();
               if (node.kind === "string") {
-                return valuePath.getParentNode().type === "heredoc"
```

Comment:
you can probably get rid of some extra logic under "string" now that we aren't printing this

Reply:
Good catch! Please check again 😄